### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by jhonatn**
Publish to from 0.7.0 to 0.8.0 and Tokamak from the "main" branch to 0.9.0. Code updated accordingly

### CODE REVIEW

File: Sources/TokamakPublish/Plot+HTML.swift [hunk 1]
```diff
@@ -17,18 +17,18 @@ import TokamakStaticHTML
...
     where V: View
   {
-    .raw(StaticHTMLRenderer(body()).html)
+    .raw(StaticHTMLRenderer(body()).render())
   }
 }
 
 extension Plot.Node: View, AnyHTML where Context == Plot.HTML.BodyContext {
-  public var innerHTML: String? {
+  public func innerHTML(shouldSortAttributes: Bool) -> String? {
```
Comment (MaxDesiatov) [inline Sources/TokamakPublish/Plot+HTML.swift:31]: Should this argument be present if it's unused? And if not, can it be changed back to a property again?

Comment (carson-katri) [reply Sources/TokamakPublish/Plot+HTML.swift:31]: I believe it has to have that argument to satisfy a requirement of [`AnyHTML`](https://github.com/TokamakUI/Tokamak/blob/main/Sources/TokamakStaticHTML/Views/HTML.swift)

Comment (MaxDesiatov) [reply Sources/TokamakPublish/Plot+HTML.swift:31]: Ah, fair enough. But then `render` on `StaticHTMLRenderer` does take `shouldSortAttributes` as an argument, so it should be passed forward here.

**Review by MaxDesiatov — CHANGES_REQUESTED** [review]

File: Sources/TokamakPublish/Plot+HTML.swift (hunk 1 above)
Comment (jhonatn) [reply Sources/TokamakPublish/Plot+HTML.swift:31]: Plot (the package being extended here) posseses no such feature as attribute sorting

Comment (MaxDesiatov) [reply Sources/TokamakPublish/Plot+HTML.swift:31]: Ah, I see, `render()` called here is coming from `Plot`, not `StaticHTMLRenderer`?

Comment (jhonatn) [reply Sources/TokamakPublish/Plot+HTML.swift:31]: That's correct

Comment (MaxDesiatov) [reply Sources/TokamakPublish/Plot+HTML.swift:31]: If so, would you be able to add a comment by that `renderer()` call clarifying that `Plot` doesn't support attribute sorting? Such comment would clarify why `shouldSortAttributes` argument is ignored, for posterity.

Comment (jhonatn) [reply Sources/TokamakPublish/Plot+HTML.swift:31]: Done!

**Review by MaxDesiatov — APPROVED** [review]
Great, thanks!

**Pull request merged**